build/mvn

```diff
@@ -36,7 +36,7 @@ _DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # Preserve the calling directory
 _CALLING_DIR="$(pwd)"
 # Options used during compilation
-_COMPILE_JVM_OPTS="-Xmx2g -XX:ReservedCodeCacheSize=1g -Xss128m"
+_COMPILE_JVM_OPTS="-Xss128m -Xmx4g -Xmx4g -XX:ReservedCodeCacheSize=128m"
```

Comment:
May I ask why you repeat `-Xmx4g` twice?

Reply:
Sorry, it copy from 

https://github.com/apache/spark/blob/865a3ded2ea1ca86be93df58205882bc509b98cd/dev/make-distribution.sh#L164

do you mind also remove the dup `-Xmx4g` from `make-distribution.sh` in this pr?

Reply:
Or maybe  we can change it to `-Xms4g`

Reply:
removed the duplicate `-Xmx4g`